bedrock/firefox/templates/firefox/includes/hub/value-cards.html

```diff
@@ -0,0 +1,69 @@
+{#
+ # I'm putting this in the includes folder for reference, though it
+ # probably wouldn't really work as an include and you might just
```

Comment:
I took a peek in Jon's branch and I can't see it, so I think we're safe to remove it. If I'm mistaken his PR should just add it back again anyway.